Automattic/wp-calypso
# Jetpack Cloud: Add isomorphic routing for jetpack-cloud-features-comparison section.

The JP Feature Comparison section need fixing, as does the earlier problem with the JP pricing section loading the default language before the actual translation. This PR precisely replicates the changes made to the price page in #72501.

https://user-images.githubusercontent.com/56598660/214801400-ca917f05-5542-48a3-b909-60e5d5acb0d5.mov



#### Proposed Changes

* Add isomorphic routing for jetpack-cloud-features-comparison section.

#### Testing Instructions

Run this branch by following the steps below (or you can use the Jetpack Cloud live link commented on this PR)
* Run `git fetch && git checkout update/jp-cloud-feature-comparison-page-isomorphic`
* Run `yarn start-jetpack-cloud`

1. Go to http://jetpack.cloud.localhost:3000/ko/features/comparison or use the Jetpack Cloud live link and append `/ko/features/comparison`.
2. Confirm that the english translations do not flash before the Korean translations.

https://user-images.githubusercontent.com/56598660/214805128-fb18c3cd-ec57-49de-8939-50320c95a076.mov



#### Pre-merge Checklist





Related to 1202858161751496-as-1203828797004526

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203828797004526
